draft-song-dns-wireformat-http-01.txt

```diff
@@ -271,17 +274,17 @@ Internet-Draft          DNS wire-format over HTTP          February 2016
 
    Proxy-DNS-Transport: xyz
```

Comment:
> I think it should be mandatory.

OK. BTW, Paul, would you like to be one of co-author with us ? or just a contributor.